### ISSUE TITLE
Make sure http verbs are of lower case. 

### ISSUE THREAD

**Issue by deepankarm** [SamWSoftware/serverless-auto-swagger#52]
`serverless.yml` allows capital verbs in `http` event. 

```yaml
  my-delete:
    name: my-delete-${self:provider.stage}
    description: Endpoint to delete
    handler: api.endpoints.DELETE
    events:
      - http: 
          method: DELETE
          path: blah/{id}
```

This will fail as only lower case verbs are supported by [openapi](https://swagger.io/docs/specification/paths-and-operations/#:~:text=OpenAPI%203.0%20supports%20get%2C%20post%2C%20put%2C%20patch%2C%20delete%2C%20head%2C%20options%2C%20and%20trace.).

**Comment by bfaulk96** [comment]
Good call, I made a PR (#53) to close this. I locked down the method types to be specific to OpenApi specifications, and made sure we are using `.toLowercase()` for defining our swagger methods

**Comment by bfaulk96** [comment]
Fixed and released as part of `2.3.0` 😄
